File: finmartserviceapi/src/main/java/magicfinmart/datacomp/com/finmartserviceapi/finmart/requestentity/ContactLeadRequestEntity.java
package magicfinmart.datacomp.com.finmartserviceapi.finmart.requestentity;

import java.util.List;

import magicfinmart.datacomp.com.finmartserviceapi.finmart.model.ContactlistEntity;

/**
 * Created by IN-RB on 17-10-2018.
 */

public class ContactLeadRequestEntity {


    /**
     * fbaid : 12
     * contactlist : [{"name":"Three","mobileno":"807232323"},{"name":"Four","mobileno":"77232323"}]
     */

    private String fbaid;
    private List<ContactlistEntity> contactlist;

    public String getFbaid() {
        return fbaid;
    }

    public void setFbaid(String fbaid) {
        this.fbaid = fbaid;
    }

    public List<ContactlistEntity> getContactlist() {
        return contactlist;
    }

    public void setContactlist(List<ContactlistEntity> contactlist) {
        this.contactlist = contactlist;
    }


}

File: finmartserviceapi/src/main/java/magicfinmart/datacomp/com/finmartserviceapi/finmart/requestentity/syncContact/ContactLeadRequestEntity.kt
package com.utility.finmartcontact.core.requestentity

import com.utility.finmartcontact.core.model.ContactlistEntity

/**
 * Created by Rajeev Ranjan on 01/04/2019.
 */
data class ContactLeadRequestEntity (
    var fbaid: String,
    var ssid : String,
    val sub_fba_id: String,
    var contactlist: List<ContactlistEntity>? = null,
    var raw_data: String
)
